### PR TITLE
Chore(entropy) add error page

### PR DESCRIPTION
**Pull request by aditya520**

### CODE REVIEW

File: pages/entropy/error-codes.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| 0xd82dd966  | AssertionFailure()          | Contract invariant failed.                                                                                                                                                                                                  |
+| 0xda041bdf  | ProviderAlreadyRegistered() | Provider already registered.                                                                                                                                                                                                |
+| 0xdf51c431  | NoSuchProvider()            | Requested Provider does not exist.                                                                                                                                                                                          |
+| 0xc4237352  | NoSuchRequest()             | Request does not exist.                                                                                                                                                                                                     |
```
Comment (m30m) [inline pages/entropy/error-codes.mdx:12]: This may also happen if it's already fulfilled

File: pages/entropy/error-codes.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+| 0x3e515085  | OutOfRandomness()           | Provider is our of committed random numbers.                                                                                                                                                                                |
```
Comment (m30m) [inline pages/entropy/error-codes.mdx:13]: typo.

File: pages/entropy/error-codes.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| 0x025dbdd4  | InsufficientFee()           | Transaction fee insufficient.                                                                                                                                                                                               |
```
Comment (m30m) [inline pages/entropy/error-codes.mdx:14]: Transaction fee -> Request fee.
Use getFee function to get the latest fee necessary for a request.

File: pages/entropy/error-codes.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+| 0xb8be1a8d  | IncorrectRevelation()       | Revelation does not match commitment.                                                                                                                                                                                       |
+| 0xb463ce7a  | InvalidUpgradeMagic()       | Governance message is invalid.                                                                                                                                                                                              |
+| 0x82b42900  | Unauthorized()              | `msg.sender` is not allowed to invoke this method.                                                                                                                                                                          |
+| 0x92555c0e  | BlockhashUnavailable()      | Blockhash is 0.                                                                                                                                                                                                             |
```
Comment (m30m) [inline pages/entropy/error-codes.mdx:18]: If entropy request was made using the `useBlockhash` flag and the reveal method is called too late, the blockhash is not available anymore and the request can not be fulfilled.

**Review by m30m — APPROVED** [review]
thanks, added some more descriptions. Some of these errors are irrelevant to the users, but I'm fine keeping them here since it's just a reference.

File: pages/entropy/error-codes.mdx (hunk 4 above)
Comment (aditya520) [reply pages/entropy/error-codes.mdx:18]: That's too verbose. That should be added in a troublshooting page.

**Pull request merged**